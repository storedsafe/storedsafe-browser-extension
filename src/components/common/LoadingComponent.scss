@import '../../variables';

.loading-component {
  display: grid;
  width: 100%;
  height: 100%;
}
