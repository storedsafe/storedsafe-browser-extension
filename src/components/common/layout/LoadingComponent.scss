@import '../../../styles/variables.scss';

.loading-component {
  display: grid;
  width: 100%;
  height: 100%;
  justify-content: center;
  align-items: center;
  box-sizing: border-box;
}
