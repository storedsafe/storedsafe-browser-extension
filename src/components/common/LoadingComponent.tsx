import React from 'react';
import { LoadingSpinner } from './LoadingSpinner';
import './LoadingComponent.scss';

export const LoadingComponent: React.FunctionComponent = () => (
  <div className="loading-component">
    <LoadingSpinner />
  </div>
);
