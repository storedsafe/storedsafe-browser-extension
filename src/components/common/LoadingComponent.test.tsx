import * as React from 'react';
import { shallow } from 'enzyme';
import { LoadingComponent } from './LoadingComponent';

test('<LoadingComponent />', () => {
  const wrapper = shallow(<LoadingComponent />);
  expect(wrapper).toMatchSnapshot();
});
