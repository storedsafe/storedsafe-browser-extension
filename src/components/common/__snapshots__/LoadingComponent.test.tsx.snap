// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingComponent /> 1`] = `ShallowWrapper {}`;
